src/main/java/de/tum/in/www1/artemis/domain/BaseExercise.java

```diff
@@ -42,6 +42,10 @@ public abstract class BaseExercise extends DomainObject {
     @JsonView(QuizView.Before.class)
     private ZonedDateTime releaseDate;
 
+    @Column(name = "start_date")
+    @JsonView(QuizView.Before.class)
+    private ZonedDateTime startDate;
```

Comment:
Why is this added to the `BaseExercise` entity? It seems to be just used for ProgrammingExercises right now? I would suggest moving it there. It is the same table anyway so no changes to the db layer are needed when we want to use it for other exercises in the futrue.

Reply:
This will be added to other exercises in the future

Reply:
I get that. But why not move it then to the top entity when it will be used for other exercises?

Reply:
If you really want to leave it like this, maybe add at least a comment `Currently only used for ....` and a ToDo to introduce it to other exercises.

Reply:
It is already working for other exercises on the server side (e.g., text exercise (server side tests)). For now it's critical for our programming exercises. Therefore, it has been implemented on the client side for that specific exercise.

Reply:
ok got it

Reply:
Added TODO for using in quiz exercise